docs/compatibility.md

```diff
@@ -0,0 +1,75 @@
+# Registry Compatibility
+
+
+## Synopsis
+*If a Manifest is pulled by _Digest_ from a Registry 2.3 with Docker Engine 1.9
+and older, and the Manifest was pushed with Docker Engine 1.10, a security check
+will cause the Engine to receive a Manifest it cannot use and the pull will fail.*
+
+## Registry Manifest Support
+
+
+Historically, the Registry has supported a [single Manifest type](https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-1.md)
+known as _Schema 2 Version 1_.
```

Comment:
I think this is confusion because we have the "v2 manifest" "schema version 1". For this document we should just talk about manifests, not "v2 manifests" and just refer as schema 1 and schema 2